recipes/r-bookdown/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set version = '0.4' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-bookdown
+  version: {{ version|replace("-", "_") }}
+
+source:
+  fn: bookdown_{{ version }}.tar.gz
+  url:
+    - https://cran.r-project.org/src/contrib/bookdown_{{ version }}.tar.gz
+    - https://cran.r-project.org/src/contrib/Archive/bookdown/bookdown_{{ version }}.tar.gz
+  sha256: 7b6323034890b7e013092cc3cf609ad7bc7e74f258e7c79bd067e51de199e1ba
+
+build:
+  number: 0
+  skip: true  # [win32]
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - r-base
+    - r-htmltools >=0.3.5
+    - r-knitr >=1.16
+    - r-rmarkdown >=0.9.6
+    - r-yaml >=2.1.14
+  run:
+    - r-base
+    - r-htmltools >=0.3.5
+    - r-knitr >=1.16
+    - r-rmarkdown >=0.9.6
+    - r-yaml >=2.1.14
+
+test:
+  commands:
+    - $R -e "library('bookdown')"  # [not win]
+    - "\"%R%\" -e \"library('bookdown')\""  # [win]
+
+about:
+  home: https://github.com/rstudio/bookdown
+  license: GPL-3
```

Comment:
Sorry! I followed examples of other r-packages on conda-forge, which were GPL, but did not have a license_file entry. I'll try to amend this.  How's this:
- I went to the CRAN bookdown page, https://cran.rstudio.com/web/packages/bookdown/index.html
- I followed the GPL-3 link given for License on that page
- I copied the License text in that link into a file LICENSE.txt that I placed in the recipe directory
- I cite this file in the meta.yaml under 'about:' as follows:
license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE.txt'
Will this suffice?
Thanks,
Bengt